webapp/src/components/loader/linear/index.tsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 
 import './styles.scss';
 
-const LinearLoader = () : JSX.Element => (
-    <div className='linear-loader'>
+const LinearLoader = ({extraClass}: {extraClass?: string}) : JSX.Element => (
+    <div className={`linear-loader ${extraClass}`}>
```

Comment:
will this class contain undefined if we don't send this prop?

Reply:
yes but that doesn't affect anything in the UI and using `? : ' '` will be more expensive code-wise

Reply:
We can initialize `extraClass = ''` in the starting to overcome both these issues.